pkg/collectors/cluster_resource_quota_test.go

```diff
@@ -44,7 +48,40 @@ func TestClusterResourceQuotaCollector(t *testing.T) {
         openshift_clusterresourcequota_labels{label_quota="test",name="quota1"} 1
 `,
 
-			MetricNames: []string{"openshift_clusterresourcequota_created", "openshift_clusterresourcequota_labels", "openshift_clusterresourcequota_usage"},
+			MetricNames: []string{"openshift_clusterresourcequota_created", "openshift_clusterresourcequota_selector", "openshift_clusterresourcequota_labels", "openshift_clusterresourcequota_usage"},
+		},
+		{
+			Obj: &v1.ClusterResourceQuota{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "quota1",
+					CreationTimestamp: metav1.Time{Time: time.Unix(1500000000, 0)},
+					Namespace:         "ns1",
+				},
+				Spec: v1.ClusterResourceQuotaSpec{
+					Quota: corev1.ResourceQuotaSpec{
+						Hard: corev1.ResourceList{
+							corev1.ResourceCPU: resource.MustParse("4.3"),
+						},
+					},
+					Selector: v1.ClusterResourceQuotaSelector {
+						AnnotationSelector : map[string]string{},
+						LabelSelector : &metav1.LabelSelector{
+								MatchLabels: map[string]string{"clusterquota": "labeltest"},
```

Comment:
Please also add a test for "MatchExpressions"

Reply:
added tests for "MatchExpressions"